app/jobs/assign_pid_job.rb

```diff
@@ -6,7 +6,7 @@ class AssignPidJob
   # This worker will connect to "h2.druid_assigned" queue
   # env is set to nil since by default the actual queue name would be
   # "h2.druid_assigned_development"
-  from_queue 'h2.druid_assigned', env: nil
+  from_queue Settings.rabbitmq.queues.druid_assigned, env: nil
```

Comment:
All of these queue changes are so that we can run multiple instances of H2 in a single environment and each has its own queue. This will allow us to run a dockerized H2 and a plain-old H2 together.